<ui version="4.0" >
 <class>HelpWidget</class>
 <widget class="QMainWindow" name="HelpWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OpenFlipper Help</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QLineEdit" name="searchEdit_" >
    <property name="geometry" >
     <rect>
      <x>500</x>
      <y>20</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton_" >
    <property name="geometry" >
     <rect>
      <x>870</x>
      <y>20</y>
      <width>77</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text" >
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton_" >
    <property name="geometry" >
     <rect>
      <x>330</x>
      <y>20</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text" >
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="forwardButton_" >
    <property name="geometry" >
     <rect>
      <x>410</x>
      <y>20</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text" >
     <string>Forward</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget" >
    <property name="geometry" >
     <rect>
      <x>9</x>
      <y>69</y>
      <width>951</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_" />
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
